core/src/SYCL/Kokkos_SYCL_Parallel_Team.hpp

```diff
@@ -304,10 +304,7 @@ class TeamPolicyInternal<Kokkos::Experimental::SYCL, Properties...>
     return std::min({
              int(m_space.impl_internal_space_instance()->m_maxWorkgroupSize),
       // FIXME_SYCL Avoid requesting to many registers on NVIDIA GPUs.
```

Comment:
Typo here that I noticed in reading the adjacent diff
```suggestion
      // FIXME_SYCL Avoid requesting too many registers on NVIDIA GPUs.
```